src/features/user/Account/History.tsx

```diff
@@ -6,6 +6,20 @@ import AccountRecord from './components/AccountRecord';
 import styles from './AccountHistory.module.scss';
 import { useRouter } from 'next/router';
 import { ProjectPropsContext } from '../../common/Layout/ProjectPropsContext';
+import { postAuthenticatedRequest } from '../../../utils/apiRequests/api';
+import { UserPropsContext } from '../../common/Layout/UserPropsContext';
+import { ErrorHandlingContext } from '../../common/Layout/ErrorHandlingContext';
+import { Alert, styled, Snackbar, CircularProgress } from '@mui/material';
+import themeProperties from '../../../theme/themeProperties';
+
+const MuiAlert = styled(Alert)({
+  '&.MuiAlert-filledSuccess': {
+    backgroundColor: themeProperties.primaryColor,
+  },
+  '.MuiAlert-message': {
+    color: 'white !important',
+  },
+});
```

Comment:
This may not be needed if you use `CustomSnackbar` instead.

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/252f1fd907426863d6a47e5c5181358fdd222211